olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -372,6 +376,10 @@ if(OLP_SDK_ENABLE_DEFAULT_CACHE)
     set(OLP_SDK_CORE_SOURCES ${OLP_SDK_CORE_SOURCES} ${OLP_SDK_CACHE_SOURCES})
 endif()
 
+if(OLP_SDK_ENABLE_DEFAULT_CACHE_LMDB)
+    # TODO: include LMDB cache implementation source files here
+endif()
```

Comment:
No dummy code. Please remove, we can add it when actually needed.

Reply:
Done